test_runner/fixtures/neon_fixtures.py

```diff
@@ -1713,17 +1718,24 @@ def init(
             if force is not None:
                 cmd.extend(["--force", force])
```

Comment:
This is probably dead code as well, this forcing, it is not parsed as far as I can see?

Reply:
It's used here: https://github.com/neondatabase/neon/blob/e6da7e29ed0a550ec59686bea7e656e16b2f13e7/test_runner/fixtures/neon_fixtures.py#L606

Reply:
It is parsed here: https://github.com/neondatabase/neon/blob/e6da7e29ed0a550ec59686bea7e656e16b2f13e7/control_plane/src/bin/neon_local.rs#L368